### ISSUE TITLE
Issue of obtaining messageId through CallbackQuery in version 6.9.7.0.

### ISSUE THREAD

**Issue by youchenga** [rubenlagus/TelegramBots#1304]
**Is your feature request related to a problem? Please describe.**

In version 6.9.7.0, I can only obtain the messageId in the callbackQuery by converting the type, as it defaults to the MaybeInaccessibleMessage type. This is somewhat inconvenient. Is there a more convenient method?

**Describe the solution you'd like**

Desire a more convenient method to obtain the messageId.

**Describe alternatives you've considered**

I'm currently using type conversion to call `val messageId = (message as Message).messageId`.

**Additional context**

```kotlin
        if (update.hasCallbackQuery()) {
            val callbackQuery = update.callbackQuery
            val message = callbackQuery.message
            val data = callbackQuery.data
            val messageId = (message as Message).messageId
            buttonTap(callbackQuery.from.id, callbackQuery.id, data, messageId)
            return
        }
```

**Comment by Amartel1986** [comment]
Same problem. By specification, this interface can be one of two classes: https://core.telegram.org/bots/api#maybeinaccessiblemessage

Both contain the following fields:
- chat
- message_id
- date

I think, all of those should be a part of the interface.

Example of a different implementation, that does that: https://pub.dev/documentation/televerse/latest/telegram/MaybeInaccessibleMessage-class.html

**Comment by Chase22** [comment]
Fixed in #1310
